### PR TITLE
test(ivy): root causes update for 'platform-browser-dynamic' and i18n tests

### PR DESCRIPTION
**Pull request by AndrewKushnir**
Updated root cause for one tests and enabled 2 tests that are now passing.

## PR Type
What kind of change does this PR introduce?

- [x] Other... Please describe: TestBed issues root causes update

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/platform-browser-dynamic/test/testing_public_browser_spec.ts [hunk 1]
```diff
@@ -63,11 +63,10 @@ if (isBrowser) {
               {providers: [{provide: FancyService, useValue: new FancyService()}]});
         });
 
-        fixmeIvy('unknown').it(
-            'provides a real ResourceLoader instance',
-            inject([ResourceLoader], (resourceLoader: ResourceLoader) => {
-              expect(resourceLoader instanceof ResourceLoaderImpl).toBeTruthy();
-            }));
+        it('provides a real ResourceLoader instance',
```
Comment (kara) [inline packages/platform-browser-dynamic/test/testing_public_browser_spec.ts]: This is still failing, see circle CI `test-ivy-aot`

**Review by kara — CHANGES_REQUESTED** [review]

File: packages/platform-browser-dynamic/test/testing_public_browser_spec.ts (hunk 1 above)
Comment (AndrewKushnir) [reply packages/platform-browser-dynamic/test/testing_public_browser_spec.ts]: Thanks for review @kara! It didn't fail locally :( Will look into it...

Comment (kara) [reply packages/platform-browser-dynamic/test/testing_public_browser_spec.ts]: It should fail locally if you are running the tests with `test_web_chromium-local`

Comment (AndrewKushnir) [reply packages/platform-browser-dynamic/test/testing_public_browser_spec.ts]: Thanks Kara, I performed further investigation, found the origin of the problem and created a new ticket (+ referenced it in this PR). Thank you.

**Review by kara — APPROVED** [review]
LGTM

**Pull request closed without merging**